tedana/decay.py

```diff
@@ -71,14 +89,18 @@ def fit_decay(data, tes, mask, masksum):
 
     if len(data.shape) == 3:
         n_samp, n_echos, n_vols = data.shape
+        fit_data = np.mean(data, axis = 2)
     else:
         n_samp, n_echos = data.shape
         n_vols = 1
 
     data = data[mask]
+    fit_data = fit_data[mask]
     t2ss = np.zeros([n_samp, n_echos - 1])
     s0vs = np.zeros([n_samp, n_echos - 1])
 
+    
+    
```

Comment:
This is nitpicky but could you delete the extra space?

Reply:
begone whitespace!